Source/Scene/OrthographicFrustum.js

```diff
@@ -286,6 +289,8 @@ define([
         update(this);
 
         //>>includeStart('debug', pragmas.debug);
+        deprecationWarning('OrthographicFrustum', 'getPixelSize is deprecated. Use getPixelDimensions instead.');
```

Comment:
Make this the very first line in the function, and not inside the `includeStart` block.